tests/devapps/XForms/XForms.Android/MainActivity.cs

```diff
@@ -62,6 +62,8 @@ protected override void OnCreate(Bundle bundle)
 
             App.UIParent = new UIParent(this, true);
```

Comment:
we can remove this now right?

Reply:
and the corresponding one on the iOS side?

Reply:
Yes, done, pushed more changes.